content/post/update-new-python-api.md

```diff
@@ -0,0 +1,160 @@
++++
+title = "How to update your Python applications to the new API"
```

Comment:
I think this needs to have "Wallaroo" in here like

How to update your Wallaroo Python application to use our new API